projects/commudle-admin/src/app/components/navbar-menu/navbar-menu.component.html

```diff
@@ -5,12 +5,36 @@
     </div>
     <div class="home-navbar-menu-option">Home</div>
   </a>
-  <a [routerLink]="['/about']" routerLinkActive="active">
+
+  <!-- <a [routerLink]="['/about']" routerLinkActive="active">
     <div class="icon">
       <fa-icon [icon]="faInfoCircle"></fa-icon>
     </div>
     <div class="home-navbar-menu-option">About</div>
+  </a> -->
+
+  <a *ngIf="mobileView" [routerLink]="['/notifications']" class="notification">
+    <div class="icon">
+      <span *ngIf="notificationCount > 0" class="notification-badge">{{ notificationCount }}</span>
+      <fa-icon [icon]="faBell"></fa-icon>
+    </div>
+    <div class="home-navbar-menu-option">Notifications</div>
+  </a>
+
+  <a *ngIf="!mobileView" class="notification">
```

Comment:
Let's think of a better way to do this

Reply:
Going with this for now, searching for other options.